awk, cut, ed, sed: add and refresh pages (#7551)
tldr-pages/tldr: pages/common/sed.md

# sed

> Edit text in a scriptable manner.
> See also: `awk`, `ed`.
> More information: <https://manned.org/sed.1>.

> - Execute commands:

Reviewer: `expression` is more technically correct than `command`.
```suggestion
- Execute an expression:
```

`sed '{{s/apple/mango/g}}'`

- Execute a script:

`sed --file={{path/to/script.sed}}`

> - Execute commands with enabled extended regular expressions:

Reviewer: Perhaps this will read better? Also, `expression` is more technically correct than `command`.
```suggestion
- Execute an expression, enabling extended regular expressions:
```

`sed --regexp-extended '{{s/(apple)/\U\1/g}}'`

> - Execute commands without automatic buffer printing:

Reviewer: `expression` is more technically correct than `command`.
```suggestion
- Execute an expression without automatic buffer printing:
```

`sed --quiet '{{1p}}'`

> - Execute commands and replace file with it's output:

Reviewer: Perhaps this will read better? Also, `expression` is more technically correct than `command`.
```suggestion
- Execute an expression and replace a file with it's output:
```
